src/styles/getMuiTheme.js

```diff
@@ -224,4 +245,11 @@ export default function getMuiTheme(baseTheme, muiTheme) {
       borderColor: palette.borderColor,
     },
   }, muiTheme);
+
+  const transformers = computedTheme.transformers ? computedTheme.transformers :
```

Comment:
Yeah, I generally agree with you. Though I suppose it's easier to change the implementation of where transformers are applied as a non breaking change when they are in an object rather than set arguments that rely on order/positioning. 

Also, since right now all of the existing arguments are being merged together, the transformers kind of break pattern a bit. I am going to put it in the arguments since you agree that it is better and we can see how @oliviertassinari feels.